src/test/bootstrap.php

```diff
@@ -12,34 +12,31 @@ class_alias('\PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase');
     class_alias('PHPUnit\Framework\Error\Warning', 'PHPUnit_Framework_Error');
 }
 
-if (!class_exists("PHPUnit_Util_ErrorHandler"))
-{
-    class PHPUnit_Util_ErrorHandler {
-        public static function handleError($errno, $errstr, $errfile, $errline)
-        {
-            $errorHandler = new \PHPUnit\Util\ErrorHandler(
-                true,
-                true,
-                true,
-                true
-            );
-
-            return $errorHandler($errno, $errstr, $errfile, $errline);
-        }
-    }
-}
-
-if (!class_exists("PHPUnit_TextUI_ResultPrinter"))
-{
-    class PHPUnit_TextUI_ResultPrinter extends \PHPUnit\TextUI\ResultPrinter {}
-}
-
+//if (!class_exists("PHPUnit_Util_ErrorHandler"))
+//{
+//    class PHPUnit_Util_ErrorHandler {
+//        public static function handleError($errno, $errstr, $errfile, $errline)
+//        {
+//            $errorHandler = new \PHPUnit\Util\ErrorHandler(
+//                true,
+//                true,
+//                true,
+//                true
+//            );
+//
+//            return $errorHandler($errno, $errstr, $errfile, $errline);
+//        }
+//    }
+//}
```

Comment:
Since we're dropping support for such old versions, we could delete all of this instead of commenting it out, unless I'm missing something.

Reply:
Yes, you are correct. I needed to cleanup a bit more.